includes/wccom-site/class-wc-wccom-site-installer.php

```diff
@@ -0,0 +1,543 @@
+<?php
```

Comment:
I'm interested in the reason for not making this part of the helper? Though it is a new feature, we still view this as the same context: Our connection to our merchants, and along with that the ability to "push" *things( any valid payload) to them given that we have a connection.

Reply:
This mostly came from team discussion (couldn't recall whether on dev roundtable or demo day, /cc @haszari @mattyza) that in general, we should avoid using term `helper` and be specific with the naming. Of course, it's tightly coupled with helper as it depends on the helper but not vice versa.